test/e2e/flowaggregator_test.go

```diff
@@ -1502,34 +1701,63 @@ func createPerftestPods(data *TestData) (podAIPs *PodIPs, podBIPs *PodIPs, podCI
 	return podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, nil
 }
 
-func createPerftestServices(data *TestData, isIPv6 bool) (svcB *corev1.Service, svcC *corev1.Service, err error) {
+func createPerftestServices(data *TestData, isIPv6 bool) (svcB *corev1.Service, svcC *corev1.Service, svcD *corev1.Service, svcE *corev1.Service, err error) {
 	svcIPFamily := corev1.IPv4Protocol
 	if isIPv6 {
 		svcIPFamily = corev1.IPv6Protocol
 	}
 
 	svcB, err = data.CreateService("perftest-b", data.testNamespace, iperfPort, iperfPort, map[string]string{"antrea-e2e": "perftest-b"}, false, false, corev1.ServiceTypeClusterIP, &svcIPFamily)
 	if err != nil {
-		return nil, nil, fmt.Errorf("Error when creating perftest-b Service: %v", err)
+		return nil, nil, nil, nil, fmt.Errorf("error when creating perftest-b Service: %v", err)
 	}
 
 	svcC, err = data.CreateService("perftest-c", data.testNamespace, iperfPort, iperfPort, map[string]string{"antrea-e2e": "perftest-c"}, false, false, corev1.ServiceTypeClusterIP, &svcIPFamily)
 	if err != nil {
-		return nil, nil, fmt.Errorf("Error when creating perftest-c Service: %v", err)
+		return nil, nil, nil, nil, fmt.Errorf("error when creating perftest-c Service: %v", err)
+	}
+
+	svcD, err = data.CreateService("perftest-d", data.testNamespace, iperfPort, iperfPort, map[string]string{"antrea-e2e": "perftest-d"}, false, false, corev1.ServiceTypeClusterIP, &svcIPFamily)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("error when creating perftest-d Service: %v", err)
+	}
+
+	svcE, err = data.CreateService("perftest-e", data.testNamespace, iperfPort, iperfPort, map[string]string{"antrea-e2e": "perftest-e"}, false, false, corev1.ServiceTypeClusterIP, &svcIPFamily)
+	if err != nil {
+		return nil, nil, nil, nil, fmt.Errorf("error when creating perftest-e Service: %v", err)
 	}
 
-	return svcB, svcC, nil
+	return svcB, svcC, svcD, svcE, nil
 }
 
 func deletePerftestServices(t *testing.T, data *TestData) {
-	for _, serviceName := range []string{"perftest-b", "perftest-c"} {
+	for _, serviceName := range []string{"perftest-b", "perftest-c", "perftest-d", "perftest-e"} {
 		err := data.deleteService(data.testNamespace, serviceName)
 		if err != nil {
 			t.Logf("Error when deleting %s Service: %v", serviceName, err)
 		}
 	}
 }
 
+func deletePerftestPod(t *testing.T, data *TestData) {
+	for _, podName := range []string{"perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"} {
+		err := data.DeletePodAndWait(defaultTimeout, podName, data.testNamespace)
+		if err != nil {
+			t.Logf("Error when deleting %s Pod: %v", podName, err)
+		}
+
+	}
+}
+
+func recreatePerftestPod(t *testing.T, data *TestData) {
```

Comment:
I have some concerns about this Pod recreation occurring in all subtests, becasuse it could extend the test duration. May I ask about the necessity of this step?

Reply:
When we check the connection in ClickHouse, we run something like this `SELECT * FROM flows WHERE (sourceIP = '%s') AND (destinationIP = '%s')`. Previously, we used the same PodIP, resulting in getting the records not for the current test. 

For this reason we need to clean the logs in ipfix and the records in ClickHouse. We can recreate test Pods to have different IPs for them or we can clean the data in IPFIX and ClickHouse. I think it is safer and easier to recreate test Pods.

Reply:
Understand. 
An alternative approach for this issue could be adding `--since-time` when fetching from IPFIX collector's log and `WHERE timeInserted > xxx` when fetching from Clickhouse. Recreating Pods is definitely easier so I don't hold a strong opinion here.

Reply:
Technically there is no guarantee that the IPs will be different when we recreate the Pods?

Maybe let's go with Yongming's idea if it works

Reply:
I've updated it to use --since-time for IPFIX collector and use flowStartSeconds to filter records in ClickHouse